### PR TITLE
Feature/025 edge cases

### PR DESCRIPTION
**Pull request by raphrel**

### CODE REVIEW

File: Tests/State/WaitingNetworkStateSpecs.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ final class WaitingNetworkStateSpecs: QuickSpec {
             }
         }
         
-        context("service is reachable") {
+        context("service is reachable ") {
```
Comment (JiceDessaint) [inline Tests/State/WaitingNetworkStateSpecs.swift]: Not sure about that

Comment (raphrel) [reply Tests/State/WaitingNetworkStateSpecs.swift]: really ?
